test/FisherPrice_Alpaka/alpaka_fisher_price.cu

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+
+//This is the main include file to give us all of the alpaka classes, functions etc.
+#include <alpaka/alpaka.hpp>
```

Comment:
We should ideally converge to a single executable that, depending on a configuration option, will be compiled against alpaka or cuda. Includes like this would the need to be handled by a higher-level one that will handle the ifdefs